src/lib/remark-plugins/rewrite-tutorial-links/__tests__/rewrite-tutorial-links.test.ts

```diff
@@ -336,7 +336,6 @@ describe('rewriteTutorialLinks remark plugin', () => {
 		const contents = await remark()
```

Comment:
It'd be good to add a test case to this file! Not totally necessary for this PR though, could be a follow up if we want to get this patch in fast.

Reply:
Do the new test cases in `src/views/product-tutorials-view/helpers/__tests__/detect-and-reformat-learn-url.test.ts` suffice?

Reply:
I think it would be best to have test cases in the rewrite plugin itself! But for this PR those updated test cases work 👌